src/s1reader/s1_orbit.py

```diff
@@ -117,7 +124,10 @@ def get_orbit_file_from_dir(path: str, orbit_dir: str) -> str:
     if not os.path.isdir(orbit_dir):
         raise NotADirectoryError(f"{orbit_dir} not found")
 
-    orbit_path = get_orbit_file_from_list(
-        path, [f'{orbit_dir}/{item}' for item in os.listdir(orbit_dir)])
+    orbit_file_list = glob.glob(os.path.join(orbit_dir, 'S1*.EOF'))
+    if len(orbit_file_list) == 0:
```

Comment:
```suggestion
    if not orbit_file_list:
```
[Utilize the implicit booleaness of the empty list](https://stackoverflow.com/a/53522).